client/me/purchases/manage-purchase/purchase-meta.tsx

```diff
@@ -419,6 +420,12 @@ function PurchaseClipboardCard( {
 					</>
 				) }
 			</div>
+			<ExternalLink
+				className="manage-purchase__license-clipboard-link"
+				href="https://jetpack.com/support/activate-a-jetpack-product-via-license-key/"
+			>
+				{ translate( 'How to activate' ) }
```

Comment:
I'm pretty sure this string is translated already, but I didn't see a specific translation so I ran it by i18n - p1723157482382979-slack-C02AED43D

Not necessarily a blocker.

Reply:
So the update from i18n is that this string is not currently translated, but was at one point, which means if we add the label `String freeze` the translation process should be relatively quick.